services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/application/ApplicationController.kt

```diff
@@ -54,7 +52,26 @@ class ApplicationController(
     private val hankeService: HankeService,
     private val disclosureLogService: DisclosureLogService,
 ) {
-    @GetMapping
+    @GetMapping("/hankkeet/{hankeTunnus}/hakemukset")
+    @Operation(
+        summary = "Get hanke applications",
+        description = "Returns list of applications belonging to a given hanke."
+    )
+    @PreAuthorize("@hankeAuthorizer.authorizeHankeTunnus(#hankeTunnus, 'VIEW')")
```

Comment:
`authorizeHankeTunnus` is a part of the base `Authorizer` class, so it can be called through any authorizer. Using the same authorizer as the other endpoints makes tests simpler.
```suggestion
    @PreAuthorize("@applicationAuthorizer.authorizeHankeTunnus(#hankeTunnus, 'VIEW')")
```

Reply:
Good to know! I changed it here and elsewhere where I had used both hankeAuthorizer and applicationAuthorizer.